### PR TITLE
[feature] Add support for store another SEPA direct debit for existing customer with Stripe 

### PR DESCRIPTION
**Pull request by dbackowski**
This will support for store another SEPA direct debit payment method for an existing customer.

### CODE REVIEW

**Comment by dbackowski** [comment]
Screenshot 2021-01-20 at 09 58 14

**Review by himn1 — APPROVED** [review]
non blocking

File: lib/active_merchant/billing/gateways/stripe.rb [hunk 1]
```diff
@@ -218,11 +218,20 @@ def store(payment, options = {})
...
-              post[:default_card] = r.params['id']
+              if options[:set_default] && r.success?
+                post[:invoice_settings] = {}
+                post[:invoice_settings][:default_payment_method] = params[:payment_method_id]
+              end
+            else
+              # The /cards endpoint does not update other customer parameters.
+              r.process { commit(:post, "customers/#{CGI.escape(options[:customer])}/cards", params, options) }
+
+              if options[:set_default] and r.success? and !r.params['id'].blank?
```
[the remark this answers is not in the document]
Comment (dbackowski) [reply lib/active_merchant/billing/gateways/stripe.rb:232]: Good catch 👍

**Review by himn1 — APPROVED** [review]
🚢

**Pull request merged**